osu.Game/Graphics/UserInterface/OsuSliderBar.cs

```diff
@@ -160,11 +161,11 @@ private void playSample(T value)
 
             var channel = sample.Play();
 
-            channel.Frequency.Value = 1 + NormalizedValue * 0.2f;
+            channel.Frequency.Value = 1 + NormalizedValue * 0.2f + RNG.NextDouble(0.02f);
             if (NormalizedValue == 0)
-                channel.Frequency.Value -= 0.4f;
+                channel.Frequency.Value -= 0.5f;
             else if (NormalizedValue == 1)
-                channel.Frequency.Value += 0.4f;
+                channel.Frequency.Value -= 0.5f;
```

Comment:
Was this really meant to be a `-=` in both cases? From this diff alone it half-looks like an error/oversight, because the signs were different before, and because the conditions aren't merged which would make sense if the two cases were to be treated in the same way.

Reply:
This was indeed intentional. Maybe worth a comment attached to it since it doesn't match expectations when reading the code.

Reply:
Was intentional, yeah. I kinda overlooked merging the conditionals as I was originally just messing around with numbers until it sounded good.